### PR TITLE
Add Christopher Allen on smart signatures

### PR DESCRIPTION
**Pull request by michaelfolkson**
Christopher is \@ChristopherA on Twitter, Stanford BPASE is \@CBRStanford on Twitter

### CODE REVIEW

**Comment by ChristopherA** [comment]
+1 I'm also @ChristopherA on Github ;-)

File: transcripts/blockchain-protocol-analysis-security-engineering/2018/2018-01-24-christopher-allen-smart-signatures.mdwn [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+We want to be able to combine these in multiple and interesting ways. For instance in a development release or a continuous integration toolchain it might have at the top level a multifactor 3-of-5 signature. One of those 5 signatures has authorized their assistant to be able to do the signing for the development release because he is on leave. Another signer has a very strong security policy that requires 2-of-2 keys to be applied to the signature where one of those keys is stored on a hardware token.
+
+# Use Case: Transactional Support
+
+Finally we want to be able to support transactions because that is one of the primary uses for authorization. Signatures are often part of a larger process. They can prove that specific transactional states exist before they proceed. We want to be able to test against various oracles. For instance we want to be able to say “No more than 5K has already been spent this month” which means it needs to know how much has already been spent by other authorizations in the history of the smart signature. There may be other uses such as provenance, a piece of art requires a transnational history in order for you to be able to purchase it.
```
Comment (ChristopherA) [inline transcripts/blockchain-protocol-analysis-security-engineering/2018/2018-01-24-christopher-allen-smart-signatures.mdwn:61]: c/transnational/transactional/

Comment (ChristopherA) [reply transcripts/blockchain-protocol-analysis-security-engineering/2018/2018-01-24-christopher-allen-smart-signatures.mdwn:61]: I read through the whole thing and that was the only typo.

**Comment by ChristopherA** [comment]
Thanks for doing this! It was one of my better talks, and despite being done over two years ago it still is quite relevant, and in fact, feels like this year the topic will coming up again, so I wanted to refer to it.

**Comment by kanzure** [comment]
https://diyhpl.us/wiki/transcripts/blockchain-protocol-analysis-security-engineering/2018/2018-01-24-christopher-allen-smart-signatures/

**Comment by michaelfolkson** [comment]
No problem @ChristopherA. Yeah it was an interesting talk especially going over the different languages people were working on early 2018. Haven't heard much from them recently apart from Simplicity. Perhaps progress stalled on the languages because of problems with the blockchains they were being built for (Ethereum, Tezos etc)

**Pull request merged**